bin/build_offload.sh

```diff
@@ -117,8 +116,12 @@ if [ "$1" != "nocmake" ] && [ "$1" != "install" ] ; then
    echo "Use ""$0 nocmake"" or ""$0 install"" to avoid FRESH START."
    echo rm -rf $BUILD_DIR/build/offload
    rm -rf $BUILD_DIR/build/offload
-   MYCMAKEOPTS="$COMMON_CMAKE_OPTS -DCMAKE_PREFIX_PATH=$ROCM_CMAKECONFIG_PATH;$AOMP_INSTALL_DIR/lib/cmake -DCMAKE_BUILD_TYPE=Release $AOMP_ORIGIN_RPATH"
-   if [ "$AOMP_LEGACY_OPENMP" == "1" ]; then
+   if [ "$AOMP_STANDALONE_BUILD" == 1 ]; then
+     MYCMAKEOPTS="$COMMON_CMAKE_OPTS -DCMAKE_PREFIX_PATH=$ROCM_CMAKECONFIG_PATH;$AOMP_INSTALL_DIR/lib/cmake -DCMAKE_BUILD_TYPE=Release $AOMP_ORIGIN_RPATH"
+   else
+     MYCMAKEOPTS="$COMMON_CMAKE_OPTS -DCMAKE_PREFIX_PATH=$ROCM_CMAKECONFIG_PATH;$INSTALL_PREFIX/lib/cmake -DCMAKE_BUILD_TYPE=Release $OPENMP_EXTRAS_ORIGIN_RPATH"
```

Comment:
```MYCMAKEOPTS``` here only has one different variable i.e(OPENMP_EXTRAS_ORIGIN_RPATH) in it for non-aomp-standalone build when compared to standalone-build in line 120. can we assign common values for MYCMAKEOPTS only once and just use.

```MYCMAKEOPTS="$COMMON_CMAKE_OPTS -DCMAKE_PREFIX_PATH=$ROCM_CMAKECONFIG_PATH;$AOMP_INSTALL_DIR/lib/cmake -DCMAKE_BUILD_TYPE=Release"```

for standalone 

```MYCMAKEOPTS="$MYCMAKEOPTS $AOMP_ORIGIN_RPATH```

for non-standalone
```MYCMAKEOPTS="$MYCMAKEOPTS $OPENMP_EXTRAS_ORIGIN_RPATH"```

Reply:
For `AOMP_STANDALONE_BUILD=0`, `AOMP_INSTALL_DIR` will be `/opt/rocm/lib/llvm` and `INSTALL_PREFIX` will be `/opt/rocm`. `INSTALL_PREFIX` is not used for `AOMP_STANDALONE_BUILD=1`.

So I want to keep `AOMP_INSTALL_DIR`  only for AOMP builds which uses `AOMP_STANDALONE_BUILD=1`.